enclave/ecalls.cpp

```diff
@@ -36,9 +36,10 @@ int enclave_XGDMatrixCreateFromFile(const char *fname, int silent, DMatrixHandle
   return XGDMatrixCreateFromFile(fname, silent, out);
 }
 
-int enclave_XGDMatrixCreateFromEncryptedFile(const char *fname, int silent, DMatrixHandle *out, char* username) {
+int enclave_XGDMatrixCreateFromEncryptedFile(const char *fnames[], bst_ulong num_files, int silent, DMatrixHandle *out, char* usernames[]) {
```

Comment:
Can you reorder the arguments so that `usernames` follows `fnames`?

Reply:
done